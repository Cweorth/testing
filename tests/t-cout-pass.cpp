#include <cut.h>

#include <iostream>

TEST(cppCout) {
    std::cout << "hello" << std::endl;
    CHECK_FILE(stdout, "hello\n");
}