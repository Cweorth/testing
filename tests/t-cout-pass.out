[  1] cppCout................................................................[1;32mOK[0m

Summary:
  tests:       1
  succeeded:   1
  skipped:     0
  failed:      0
